frontend/src/scenes/error-tracking/queries.ts

```diff
@@ -124,14 +132,67 @@ export const errorTrackingGroupQuery = ({
 }): ErrorTrackingQuery => {
     return {
         kind: NodeKind.ErrorTrackingQuery,
-        eventColumns: ['uuid', 'properties', 'timestamp', 'person'],
         fingerprint: fingerprint,
         dateRange: dateRange,
         filterGroup: filterGroup as PropertyGroupFilter,
         filterTestAccounts: filterTestAccounts,
     }
 }
 
+export const errorTrackingGroupEventsQuery = ({
+    select,
+    fingerprints,
+    dateRange,
+    filterTestAccounts,
+    filterGroup,
+    offset,
+}: {
+    select: string[]
+    fingerprints: ErrorTrackingGroup['fingerprint'][]
+    dateRange: DateRange
+    filterTestAccounts: boolean
+    filterGroup: UniversalFiltersGroup
+    offset: number
+}): EventsQuery => {
+    const group = filterGroup.values[0] as UniversalFiltersGroup
+    const properties = group.values as AnyPropertyFilter[]
+
+    const where = [
+        `has(${stringifyFingerprints(
+            fingerprints
+        )}, JSONExtract(ifNull(properties.$exception_fingerprint,'[]'),'Array(String)'))`,
+    ]
```

Comment:
I really don't like writing HogQL on the frontend and having to parse it on the backend